src/OpenTelemetry.AutoInstrumentation.Native/clr_helpers.cpp

```diff
@@ -610,6 +617,10 @@ WSTRING GetSigTypeTokName(PCCOR_SIGNATURE& pbCur, const ComPtr<IMetaDataImport2>
     {
         tokenName += WStr("&");
     }
+    if (pointer_flag)
```

Comment:
nit: formatting error or intentional? 
missing blank lines before `if` and before `return`.

Reply:
Added lines in 0ca789cd5957f17546dbfed2fca0b2732dee6490

Technically it was passing native code check. There is not strict rule for this.